backend/penndata/management/commands/get_events.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+from urllib.parse import urljoin
+
+from bs4 import BeautifulSoup
+from django.core.management.base import BaseCommand
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from penndata.models import Event
+
+
+PENN_TODAY_WEBSITE = "https://penntoday.upenn.edu/events"
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        now = datetime.datetime.now()
```

Comment:
nit: Can we change this to django `timezone.localtime()` so we are more precise with our datetimes (local vs utc)